File: mycollab-services-ondemand/src/main/java/com/mycollab/ondemand/module/support/domain/SimpleBillingAccount2.java
package com.mycollab.ondemand.module.support.domain;

import com.mycollab.module.user.domain.BillingAccount;
import com.mycollab.module.user.domain.SimpleUser;

import java.util.Date;
import java.util.List;

/**
 * @author MyCollab Ltd
 * @since 5.2.8
 */
public class SimpleBillingAccount2 extends BillingAccount {
    private int numUsers;
    private int numProjects;
    private Date lastAccessTime;
    private List<SimpleUser> accountOwners;

    public int getNumUsers() {
        return numUsers;
    }

    public void setNumUsers(int numUsers) {
        this.numUsers = numUsers;
    }

    public int getNumProjects() {
        return numProjects;
    }

    public void setNumProjects(int numProjects) {
        this.numProjects = numProjects;
    }

    public Date getLastAccessTime() {
        return lastAccessTime;
    }

    public void setLastAccessTime(Date lastAccessTime) {
        this.lastAccessTime = lastAccessTime;
    }

    public List<SimpleUser> getAccountOwners() {
        return accountOwners;
    }

    public void setAccountOwners(List<SimpleUser> accountOwners) {
        this.accountOwners = accountOwners;
    }
}

File: mycollab-services-ondemand/src/main/java/com/mycollab/ondemand/module/billing/domain/SimpleBillingAccount2.kt
package com.mycollab.ondemand.module.billing.domain

import com.mycollab.module.user.domain.BillingAccount
import com.mycollab.module.user.domain.SimpleUser

import java.util.Date

/**
 * @author MyCollab Ltd
 * @since 5.2.8
 */
class SimpleBillingAccount2 : BillingAccount() {
    var numUsers: Int = 0
    var numProjects: Int = 0
    var lastAccessTime: Date? = null
    var accountOwners: List<SimpleUser>? = null
}
